pinot-broker/src/main/java/org/apache/pinot/broker/requesthandler/BaseBrokerRequestHandler.java

```diff
@@ -762,23 +763,31 @@ private void handleSubquery(Expression expression, long requestId, JsonNode json
   /**
    * Resolves the actual table name for:
    * - Case-insensitive cluster
-   * - Table name in the format of [database_name].[table_name]
+   * - Table name in the format of [database_name].[namespace].[table_name]
```

Comment:
I don't think this change is required. The `[namespace].[table_name]` is the table name, and within pinot there is no concept of `namespace`